packages/react/src/patterns/sub-patterns/ContentBlock/ContentBlock.js

```diff
@@ -55,10 +56,58 @@ const ContentBlock = ({ heading, copy, children, customClassName, cta }) => {
         {children}
       </div>
       {cta && _renderCTA(cta)}
+    </>
+  );
+
+  const title = (
+    <div>
+      {heading && (
+        <h2
+          data-autoid={`${stablePrefix}--content-block__heading`}
+          className={`${prefix}--content-block__heading`}>
+          {heading}
+        </h2>
+      )}
+    </div>
+  );
+
+  return (
+    <div
+      data-autoid={`${stablePrefix}--content-block`}
+      className={cx(`${prefix}--content-block`, customClassName)}>
+      {linkList
+        ? _layoutWrap(
+            <>
+              {title}
+              <div></div>
+            </>
+          )
+        : title}
+      {linkList
+        ? _layoutWrap(
+            <>
+              <div>{content}</div>
+              <div>
+                <LinkList {...linkList} />
+              </div>
+            </>
+          )
+        : content}
     </div>
   );
 };
 
+/**
+ * wraps content in layout component
+ *
+ * @private
+ * @param {Element} content content elements
+ * @returns {*} jsx cta component
+ */
+const _layoutWrap = content => (
+  <Layout type="2-1">{content.props.children}</Layout>
```

Comment:
I believe the nested option would need to be passed in to make this work. Probably should also allow for other props like border to be turned on as well.

Reply:
done